src/node_constants.cc

```diff
@@ -1237,6 +1237,19 @@ void DefineCryptoConstants(Local<Object> target) {
   NODE_DEFINE_STRING_CONSTANT(target,
                               "defaultCipherList",
                               per_process_opts->tls_cipher_list.c_str());
+
+  Environment* env = Environment::GetCurrent(target->GetIsolate());
+
+  if (env->options()->tls_v1_0)
+    NODE_DEFINE_STRING_CONSTANT(target, "defaultMinTLSVersion", "TLSv1");
+  else if (env->options()->tls_v1_1)
+    NODE_DEFINE_STRING_CONSTANT(target, "defaultMinTLSVersion", "TLSv1.1");
+  else
+    NODE_DEFINE_STRING_CONSTANT(target, "defaultMinTLSVersion", "TLSv1.2");
```

Comment:
Can we do this in JS land, using `getOptionValue()` from `require('internal/options')`?

Reply:
done, PTAL

Reply:
@sam-github great, thanks!